cmd/forseti/main.go

```diff
@@ -70,6 +71,7 @@ func GetConfig() (Config, error) {
 	pflag.Duration("equipments-refresh", 30*time.Second, "time between refresh of equipments data")
 	pflag.String("free-floatings-uri", "", "format: [scheme:][//[userinfo@]host][/]path")
 	pflag.String("free-floatings-token", "", "token for free floating source")
+	pflag.Bool("free-floatings-active", true, "activation of vehicles in Fluctéo data")
```

Comment:
With the parameter name "free-floatings-active", I expect to have no free-floating data when this parameter is set to "false". But I would still get some data.
So I suggest some renaming
```suggestion
	pflag.Bool("free-floatings-refresh-active", true, "activate the periodic refresh of Fluctuo data")
```

Reply:
:+1: